 
onega@ub-laptop:~/testqt1$ ./testqt1 test.py
usage: ./testqt1 <python file path>
main(2, ./testqt1)
/home/onega/testqt1/testqt1
python_exec_file exec_file test.py
sys.stdout encoding is "UTF-8"
__main__, site, io, encodings, copyreg, _weakref, abc, builtins, posixpath, errno, encodings.latin_1, genericpath, stat, zipimport, _codecs, encodings.utf_8, sys, codecs, os.path, sitecustomize, signal, _io, _weakrefset, posix, encodings.aliases, os, _abcoll
sys.path=/usr/lib/python3.1, /usr/lib/python3.1/plat-linux2, /usr/lib/python3.1/lib-dynload, /usr/lib/python3.1/dist-packages, /usr/local/lib/python3.1/dist-packages
python_exec_file call MyFunc return some_args from python3.1.1+ (r311:74480, Nov  2 2009, 15:07:08)
[GCC 4.4.1] on linux2
